assets/app/scripts/services/logLinks.js

```diff
@@ -50,19 +50,42 @@ angular.module('openshiftConsole')
         newTab(params);
       };
 
-      // Not currently used.
-      // new tab: path/to/current?view=textonly
-      var textOnlyLink = function() {
-        newTab({view: 'textonly'});
+      // Kibana archive URL v1: using a string & _.template
+      var archiveTpl = [
+        '<%= baseurl %>',
+        '/#/discover?',
+        '_g=',
+        '(refreshInterval:(display:Off,pause:!f,section:0,value:0),',
+        'time:(from:now-7y,mode:quick,to:now))&_a=(columns:!(_source),',
+        'filters:!((meta:(disabled:!f,index:\'<%= namespace %>.*\',key:kubernetes_namespace_name,negate:!f,value:<%= namespace %>),',
+        'query:(match:(kubernetes_namespace_name:(query:<%= namespace %>,type:phrase)))),',
+        '(meta:(disabled:!f,index:\'<%= namespace %>.*\',key:kubernetes_pod_name,negate:!f,value:<%= podname %>),',
+        'query:(match:(kubernetes_pod_name:(query:<%= podname %>,type:phrase))))),',
+        'index:\'<%= namespace %>.*\',interval:auto,query:\'\',sort:!(time,desc))'
+      ].join('');
```

Comment:
at least use double quotes so you don't have to escape single quotes, and possibly format it so we can check at a glance that parens are matched?
```
"?_g=(",
  "refreshInterval:(...),",
  "time:(...)",
")",
"&_a=(",
  "columns:!(_source),",
  "filters:!(...),",
...
")",
```